sites/fast-website/src/app/components/background-design/background-design.ts

```diff
@@ -82,6 +85,21 @@ export class BackgroundDesign extends FASTElement {
             this.prevPerf = perf;
         };
         performAnimation(window.performance.now());
+
+        this.setupFadeObserver();
+    }
+
+    setupFadeObserver() {
+        const observer = new IntersectionObserver(
+            entries => {
+                entries.forEach(entry => (this.faded = !entry.isIntersecting));
+            },
+            { threshold: [0.5] }
+        );
+
+        const heroSection: HTMLElement = document.getElementById("hero") as HTMLElement;
```

Comment:
I don't have a good proposal, but curious if there are any issue with referencing this (assuming it's in another elements shadow DOM?)...

Reply:
It's actually an element in the actual DOM. Since the background itself doesn't scroll, we need to observe something that does.

Reply:
party, if it's in the light dom we are good :thumbsup: